<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arrayDlg</class>
 <widget class="QWidget" name="arrayDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>arrayDlg</string>
  </property>
  <widget class="QScrollArea" name="m_scrollArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>3</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="m_arraydlgWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>498</width>
      <height>398</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_ok">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>415</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancel">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>415</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
